src/components/Books.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './styles/books.css';
+
+const Books = () => (
+  <>
+    <div className="books">
+      <form action="">
+        <input type="text" placeholder="Add title" />
+        <input type="text" placeholder="Add Author" />
+        <select name="category" id="category">
+          <option value="history">History</option>
+          <option value="english">English</option>
+          <option value="economy">Economy</option>
+          <option value="literature">Literature</option>
+        </select>
+      </form>
+    </div>
+  </>
+);
```

Comment:
- [ ] In this component right now you are just displaying HTML/JSX code it doesn't really function as intended. the books component should render the `Book` and `CreateBook` components.
- the `Book` component takes the book data as a prop and renders the title and the author and has a button for removing books
- the `CreateBook` component renders a form with inputs for the title and the author of the book and has a button to submit
Both the button for removing books and for submitting the form should be placeholder buttons right now they don't have to do anything.